homework/index.js

```diff
@@ -30,15 +32,126 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
+  function renderContributors(contributors, contributorsContainer) {
+    createAndAppend('p', contributorsContainer, {
+      text: 'Contributors',
+      class: 'contributor-header',
+    });
+    contributors.forEach(contributor => {
+      const contributorList = createAndAppend('ul', contributorsContainer, {
+        class: 'contributor-list',
+      });
+
+      const contributorListItem = createAndAppend('li', contributorList);
+
+      const anchorLink = createAndAppend('a', contributorListItem, {
+        href: contributor.html_url,
+        target: '_blank',
+        class: 'contributor-item',
+      });
+
+      createAndAppend('img', anchorLink, {
+        src: contributor.avatar_url,
+        alt: contributor.login,
+        class: 'image',
+      });
+
+      const contributorDetails = createAndAppend('div', anchorLink, {
+        class: 'contributor-data',
+      });
+      createAndAppend('div', contributorDetails, { text: contributor.login });
+      createAndAppend('div', contributorDetails, {
+        text: contributor.contributions,
+        class: 'contributor-badge',
+      });
+    });
+  }
+
+  function fetchAndRenderData(selectedRepo, repoContainer, contributorsContainer, root) {
+    repoContainer.innerHTML = '';
+    contributorsContainer.innerHTML = '';
+
+    const updatedAt = new Date(selectedRepo.updated_at);
```

Comment:
This variable is unused and produces an ESLint error (red underlined in VSCode). You should fix ESLint errors before sending in a PR.

Note that the repository date as displayed on the page is not a user-friendly format. This variable was probably intended to fix that but is left unused. Please pay attention to detail.

Reply:
Hi @remarcmij, 
Thanks for the feedback and for your time. Regarding all the issues  I am still working on them. I should have stated that in the pull request. 
I could have probably received this feedback on the week1 or week2 homework if I had used this format earlier. The week3 is like an overhaul of week1 & 2 because I want to make it look better. But I will fix the issues and let you know once it's all done.